scripts/lib/k8s.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+ensure_controller_gen() {
+  go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0
+}
+
+ensure_service_controller_running() {
+  echo "Ensure that service controllers are running for given version"
+  # Give executable permission
+  chmod +x ./scripts/generate-crds.sh
+
+  __image_version=$1
+  __crd_path=$2
+
```

Comment:
I was expecting this function to have a `service` parameter, so that it only ensured a controller for a specific service (passed as a parameter) was running... perhaps you could move the `for` loop across the `services/` directory below outside this function and make this function smaller-scoped?